pages/en/billing.mdx

```diff
@@ -4,7 +4,7 @@ title: Billing on Subgraph Studio
 
 ## Billing on Arbitrum
 
```

Comment:
I see that in a [past version](https://github.com/graphprotocol/docs/pull/242/commits/ed47d7eb1174bed65562b2a9743abb712145501b) (Line 11) provides a general explanation that The Graph protocol lives on Mainnet with the billing balance on Polygon (now Arbitrum). Will we still have that? If not, I would recommend that we add that line back in at the beginning to give that context. If it is already there, feel free to ignore.

Reply:
Added that back in there - thank you!